CHANGES.rst

```diff
@@ -178,8 +223,9 @@ Bug Fixes
 
 - ``astropy.coordinates``
 
-  - Changed obsgeoloc and obsgeovel frame attributes to CartesianRepresentation. Fixes creation of
-    GCRS frames representing a location on Earth with multiple obstimes. [#5207]
+  - GCRS frames representing a location on Earth with multiple obstimes are now allowed. This means that
+   the solar system routines ``get_body``, ``get_moon`` and ``get_sun`` now work with non-scalar times and
```

Comment:
I think the travis failure is because the indentation here is not exactly right (add one more space to align this line and the one below).